packages/api-client/src/components/ApiClient/Response/ResponseBody.vue

```diff
@@ -1,39 +1,42 @@
 <script lang="ts" setup>
 import { ScalarCodeBlock } from '@scalar/components'
+import { isJsonString } from '@scalar/oas-utils'
+import type { AxiosHeaders } from 'axios'
 import contentType from 'content-type'
-import { computed } from 'vue'
+import { computed, toRaw } from 'vue'
 
 import { CollapsibleSection } from '../../CollapsibleSection'
 
-const props = withDefaults(
-  defineProps<{
-    active: boolean
-    data: any
-    headers: Record<string, string>[]
-  }>(),
-  {
-    active: false,
-    data: null,
-  },
-)
+const props = defineProps<{
+  response?: any
+}>()
 
+// In order to render the response body, we need to know the media type.
 const mediaType = computed(() => {
-  const contentTypeHeader = props.headers.find(
-    (header) => header.name.toLowerCase() === 'content-type',
+  // Transform all header keys to lowercase
+  const headers = Object.fromEntries(
```

Comment:
Would be nice to see a `normalizeHeaders` utility function here that we can share.